src/test/java/org/pesmypetcare/webservice/dao/GoogleCalendarDaoTest.java

```diff
@@ -0,0 +1,171 @@
+package org.pesmypetcare.webservice.dao;
+
+import com.google.api.services.calendar.model.Calendar;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.Events;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.pesmypetcare.webservice.error.CalendarAccessException;
+import org.pesmypetcare.webservice.firebaseservice.CalendarServiceFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.ArgumentMatchers.same;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verify;
+
+/**
+ * @author Marc Simó
+ */
+@ExtendWith(MockitoExtension.class)
+public class GoogleCalendarDaoTest {
+    private static String accessToken;
+    private static String calendarId;
+    private static String eventId;
+    private static Calendar calendar;
+    private static Event event;
+    private static Events eventsModel;
+    private static List<Event> eventList;
+
+    @Mock
+    private CalendarServiceFactory factory;
+    @Mock
+    private com.google.api.services.calendar.Calendar service;
+    @Mock
+    private com.google.api.services.calendar.Calendar.Calendars calendars;
+    @Mock
+    private com.google.api.services.calendar.Calendar.Calendars.Insert insertCalendar;
+    @Mock
+    private com.google.api.services.calendar.Calendar.Calendars.Delete deleteCalendar;
+    @Mock
+    private com.google.api.services.calendar.Calendar.Events events;
+    @Mock
+    private com.google.api.services.calendar.Calendar.Events.List eventsList;
+    @Mock
+    private com.google.api.services.calendar.Calendar.Events.Delete deleteEvent;
+    @Mock
+    private com.google.api.services.calendar.Calendar.Events.Update updateEvent;
+    @Mock
+    private com.google.api.services.calendar.Calendar.Events.Get getEvent;
+    @Mock
+    private com.google.api.services.calendar.Calendar.Events.Insert insertEvent;
+
+    @InjectMocks
+    private GoogleCalendarDao googleCalendarDao = new GoogleCalendarDaoImpl();
+
+    @BeforeAll
+    public static void setUp() {
+        accessToken = "tokenparatests";
+        calendarId = "calendarId";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "calendarId" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5403404)